packages/components/tooltip/src/tooltip.css

```diff
@@ -0,0 +1,61 @@
+:host {
+	position: relative;
+	cursor: pointer;
+	&:before {
+		position: absolute;
+		display: inline-block;
+		align-items: center;
+		background: var(--ts-color-black);
+		border-radius: var(--ts-radius);
+		color: var(--ts-color-white);
+		padding: 0 var(--ts-unit-half);
+		height: var(--ts-unit-double);
+		line-height: var(--ts-unit-double);
+		content: attr(tooltip);
+		justify-content: center;
+		opacity: 0;
+		overflow: hidden;
+		pointer-events: none;
+		z-index: var(--ts-zindex-max);
+		transition: var(--ts-transition-fast) ease opacity;
+		transition-delay: var(--ts-transition-fast);
```

Comment:
`--ts-transition-fast` is a little bit misleading for me, especially using it in `transition-delay`. 
Is it okay to add `-duration` or `-time` at the end of the variable name?

Reply:
Or adding another variable `--ts-transition-delay-short`?

Reply:
I copy the `variables` from `tradeshift.ui`. It is better keep them consistent.
May be `--ts-transition-fast`  is a bad name. It is for Animations.